easybuild/easyconfigs/c/CGAL/CGAL-4.0-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,25 @@
+name = 'CGAL'
+version = '4.0'
+
```

Comment:
minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!